src/index.html

```diff
@@ -34,15 +24,19 @@
       </a>
       <nav class="nav">
         <ul class="nav__list">
+
```

Comment:
nope, those are elements of different levels of nesting

![image](https://user-images.githubusercontent.com/103033758/233121761-3d4d2450-ec03-418c-b858-de0faa97c99d.png)


```
<parent>
  <child></child> // no new line before, different nesting levels
     --------->      // new line can be here, it is between elements of the same nesting level
  <child></child>
     ----------> // same nesting level
  <child></child> // can't add, different levels
</parent>
```

Reply:
important thing:
it is not always necessary to add space between elements on the same level. But if you see that you should improve readability - it is the case